linalg/ode.cpp

```diff
@@ -407,6 +417,20 @@ void AdamsBashforthSolver::Init(TimeDependentOperator &f_)
 
```

Comment:
The % sign in `idx[i] = (smax-i)%smax;` is unnecessary right, this is just ordering the idx backwards, idx[0]=smax,...,idx[smax-1]=1?

Reply:
For i=0 this is not the case, though.
We could also consider `idx[i] = smax-i -1`.
Perhaps that is nicer then the current.
This could perhaps better be done in #3480